roles/ssp/tasks/configure-common.yml

```diff
@@ -26,6 +26,41 @@
   tags:
     - ssp:config:authsources
 
+
+
+
+- name: Copy SP SSL certificate (.crt)
+  copy:
+    content: '{{ item.sp_ssl_certificate }}'
+    dest: "{{ ssp_certdir }}/sp-sso.crt"
```

Comment:
Change `sp-sso.crt` to `sp-{{ item.name }}.crt`. See also https://github.com/rciam/rciam-deploy/blob/5e4c2b009e4c6d7a5db3becd0f8584a2c9497a8b/roles/ssp/tasks/configure-common.yml#L19